src/hooks/useDynamicImport.ts

```diff
@@ -8,7 +8,7 @@ export const useDynamicImport = (name: string): Props => {
         let resolved = false;
 
         import(`@docgen/${name}.json`)
-            .then(props => {
+            .then(({ props }) => {
```

Comment:
I thought it was easier to update the component's `createData` with the whole metadata, not only the props.